testapp/tests/django/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from django.db import transaction
+
+
+@pytest.fixture(autouse=True)
+def enable_db_access_for_all_tests(db):
+    """
+    without this, tests (including old-style) have to explicitly declare db as a dependency
+    https://pytest-django.readthedocs.io/en/latest/faq.html#how-can-i-give-database-access-to-all-my-tests-without-the-django-db-marker
+    """
+    pass
+
+
+@pytest.fixture(scope="session")
+def globally_scoped_fixture_helper(django_db_setup, django_db_blocker):
+    with django_db_blocker.unblock():
+        # Wrap in try + atomic block to do non crashing rollback
+        # This means we don't have to re-create a test DB each time
+        try:
+            with transaction.atomic():
+                yield
+                raise Exception
+        except Exception:
+            pass
```

Comment:
We're now sorta testing through django rather than pytest directly. This is the standard root conftest fixture in every project